saleor/static/dashboard-next/products/components/ProductAvailabilityForm/ProductAvailabilityForm.tsx

```diff
@@ -16,7 +16,10 @@ import i18n from "../../../i18n";
 const styles = (theme: Theme) =>
   createStyles({
     date: {
-      marginTop: theme.spacing.unit
+      "& svg": {
```

Comment:
Isn't it a copy of `saleor/static/dashboard-next/components/VisibilityCard/VisibilityCard.tsx`?